### ISSUE TITLE
can you explain more detail about features preprocessing algorithm

### ISSUE THREAD

**Issue by bewithme** [felixchenfy/Realtime-Action-Recognition#72]
can you explain more detail about features preprocessing algorithm in
s3_preprocess_features.py ?

**Comment by felixchenfy** [comment]
No. I just pick the window_size based on my experience --- I picked 0.5s because I think as a human I can recognize other people's action from a 0.5s video.

It's not related to LSTM at all. This project doesn't use LSTM
